zeppelin-zengine/src/main/java/org/apache/zeppelin/helium/HeliumBundleFactory.java

```diff
@@ -129,6 +132,7 @@ void installNodeAndNpm() throws TaskRunnerException {
       YarnInstaller yarnInstaller = frontEndPluginFactory.getYarnInstaller(getProxyConfig());
       yarnInstaller.setYarnVersion(YARN_VERSION);
       yarnInstaller.install();
+      yarnCacheDir.mkdirs();
```

Comment:
This allows me to have unit tests passing. Raised on https://issues.apache.org/jira/browse/ZEPPELIN-2469, pull request https://github.com/apache/zeppelin/pull/2362